src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -0,0 +1,1150 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
My suggestion wasn't to make the type public. It was to put a shared copy of the single source file under src\Common\ and then include it in both .csproj projects.